examples/analysis-templates.yaml

```diff
@@ -36,6 +36,7 @@ metadata:
 spec:
   metrics:
   - name: random-fail
+    count: 2
```

Comment:
is this change intentional?

Reply:
Yes, I've put the count there as there is an error during applying rollout
> AnalysisTemplate random-fail has metric random-fail which runs indefinitely